meson.build

```diff
@@ -409,107 +412,116 @@ bdb_major_version = ''
 bdb_minor_version = ''
 bdb_version = ''
 
-bdb_dirs = [
-    '/usr/local',
-    '/usr/pkg',
-    '/opt/local',
-    macos_prefix / 'opt/berkeley-db',
-    '/usr',
-]
+if bdb_path != ''
+    bdb_dirs = [ bdb_path ]
+else
+    bdb_dirs = [
+        '/usr/local',
+        '/usr/pkg',
+        '/opt/local',
+        macos_prefix / 'opt/berkeley-db',
+        '/usr',
+    ]
+endif
+
 bdb_subdirs = [
     'db4.6',
-    'db4.7',
-    'db4.8',
-    'db4',
     'db46',
+    'db4.7',
     'db47',
+    'db4.8',
     'db48',
+    'db4',
     'db5.0',
+    'db50',
     'db5.1',
+    'db51',
     'db5.2',
     'db5.3',
     'db5',
-    'db50',
-    'db51',
     'db6.1',
     '',
```

Comment:
Do you want to append `db6.2` for good measure, since this is the later version you have?